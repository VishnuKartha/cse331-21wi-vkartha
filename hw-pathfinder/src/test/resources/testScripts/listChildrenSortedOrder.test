# testing to make sure that the children are listed in alphabetical order and if
# there are two edges for the same child in listChildren they should be ordered alphabetically
# by edge.

# Create a Graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Adds 4 edges
AddEdge graph1 n1 n3 3.0
AddEdge graph1 n1 n4 4.0
AddEdge graph1 n1 n2 2.0
AddEdge graph1 n1 n2 1.0

#Lists the children
ListChildren graph1 n1