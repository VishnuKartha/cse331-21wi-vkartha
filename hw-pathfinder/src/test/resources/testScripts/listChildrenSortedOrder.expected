# testing to make sure that the children are listed in alphabetical order and if
# there are two edges for the same child in listChildren they should be ordered alphabetically
# by edge.

# Create a Graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Adds 4 edges
added edge 3.000 from n1 to n3 in graph1
added edge 4.000 from n1 to n4 in graph1
added edge 2.000 from n1 to n2 in graph1
added edge 1.000 from n1 to n2 in graph1

#Lists the children
the children of n1 in graph1 are: n2(1.0) n2(2.0) n3(3.0) n4(4.0)